package.json

```diff
@@ -1,6 +1,12 @@
 {
   "name": "root",
   "private": true,
+  "workspaces": [
+    "packages/*"
```

Comment:
There is the `ethereumjs-testing` submodule implicitly included here, might this have any side effects?

Reply:
that's a great point, it does seem to be ignoring it so far as a workspace.

when running commands at root with `--workspaces` the command is not executed in `ethereumjs-testing`.